huxley/api/views/register.py

```diff
@@ -50,6 +59,59 @@ def register(self, request, *args, **kwargs):
             registration_serializer.is_valid(raise_exception=True)
             registration_serializer.save()
 
+        if Conference.get_current().invoicing_enabled and handler is not None:
+            school_data = user_data['school']
+            address = Address(
+                line1=school_data['address'],
+                line2='',
```

Comment:
is school's address guaranteed to be in only 1 line?

like for example if they have a particular room number for the teacher's classroom, how is that handled?

Reply:
I originally implemented Address with two address lines, but it wasn't until I integrated with `register.py` that I realized that Huxley itself only gives one line for street addresses